static/app/components/featureBadge.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import {Fragment} from 'react';
+
+import FeatureBadge from 'sentry/components/featureBadge';
+import Matrix from 'sentry/components/stories/matrix';
```

Comment:
Is `Matrix` used outside of stories? I would expect this to live in `sentry/stories/components` if not

Reply:
It's probably not going to be used outside of a story ever, idk what product would include it.

We don't have that top-level `static/app/$domain` convention yet. Looking at it, things like `app/plugins` seems a little lost right now, there's even a TSC task to move stuff into components: https://github.com/getsentry/frontend-tsc/issues/37 (previously: https://www.notion.so/sentry/Move-PluginIcon-to-components-326f1174b05247d68e266bbcd46932ac). So i'm not really sure tbh, i'd rather label `app/plugins` as the outlier, and stick with components instead of creating a 2nd exception for now.